contributed_definitions/nyaml/README.md

```diff
@@ -0,0 +1,75 @@
+# Application Definition for SPM Domain
```

Comment:
Why not add this to the manual instead?

Reply:
Thanks for notifying that. I will remove it.

Reply:
Removed.